src/api/Snapshot.ts

```diff
@@ -455,15 +455,15 @@ export class Snapshot extends API {
   }
 
   fetchProposals = async (
-    params: { start: Date; end: Date; fields: (keyof SnapshotProposal)[] },
+    params: { start: Date; end: Date; orderBy?: string; scoresState?: string; fields: (keyof SnapshotProposal)[] },
```

Comment:
I think it would be good to constrain the values that scoresState can take instead of 'string' 🤔